### ISSUE TITLE
GTK Accessibility relations

### ISSUE THREAD

**Issue by ZettaScript** [muriloventuroso/pdftricks#66]
When using a screen reader (e.g. Orca), you can only navigate between the form items, but cannot read the labels. This is a problem for visually impaired people.

Accessibility relations can be added so that the screen reader known which label corresponds to what focusable item.

I'll try to make a PR about this. I know how to do it with GTK but don't have experience in Vala yet.

**Comment by ZettaScript** [comment]
In fact the version 0.4.1 is partially accessible but master is completely not: Orca says nothing. I'll see how to fix it...
